pyproject.toml

```diff
@@ -42,7 +42,7 @@ multipledispatch = "0.6.0"
 nameparser = "1.1.0"  # nameparser is for author name manipulations
 nltk = "3.6.7"  # nltk is a textblob dependency.
 oauth2client = "4.1.3"  # Deprecated and should be replaced.
-palace-webpub-manifest-parser = "~2.0.3"
+palace-webpub-manifest-parser = { git = "https://github.com/ThePalaceProject/webpub-manifest-parser.git", branch = "feature/py310" }
```

Comment:
Was this used for testing? Is there a py 3.10-compatible version of `palace-webpub-manifest-parser` up on PyPI that we can use here, instead?

Reply:
Yup the webpub PR was still in progress yesterday. It is updated now in PyPi Jonathan